docs/minio-client-complete-guide.md

```diff
@@ -23,23 +23,64 @@ version  Print version info.
 ```
 
 ## 1.  Download Minio Client
+### Docker Stable
+```
+docker pull minio/mc
+docker run minio/mc ls play
+```
 
-| Platform | Architecture | URL |
-| ---------- | -------- |------|
-|GNU/Linux|64-bit Intel|https://dl.minio.io/client/mc/release/linux-amd64/mc|
-|Apple OS X|64-bit Intel|https://dl.minio.io/client/mc/release/darwin-amd64/mc|
-|Microsoft Windows|64-bit|https://dl.minio.io/client/mc/release/windows-amd64/mc.exe|
+### Docker Edge
+```
+docker pull minio/mc:edge
+docker run minio/mc:edge ls play
+```
+
+**Note:** Above examples run `mc` against Minio [_play_ environment](#test-your-setup) by default. To run `mc` against other S3 compatible servers, start the container this way:
+
+```sh
+docker run -it --entrypoint=/bin/sh minio/mc
+```
+
+then use the [`mc config` command](#add-a-cloud-storage-service).
 
-### Install from Homebrew
-Install minio packages using [Homebrew](http://brew.sh/) 
+### Homebrew (macOS)
+Install mc packages using [Homebrew](http://brew.sh/)
 
 ```sh
 brew install minio/stable/mc
 mc --help
 ```
 
+### Binary Download (GNU/Linux)
+| Platform | Architecture | URL |
+| ---------- | -------- |------|
+|GNU/Linux|64-bit Intel|https://dl.minio.io/client/mc/release/linux-amd64/mc |
+
+```sh
+chmod +x mc
+./mc --help
+```
+
+### Binary Download (Microsoft Windows)
+| Platform | Architecture | URL |
+| ---------- | -------- |------|
+|Microsoft Windows|64-bit|https://dl.minio.io/client/mc/release/windows-amd64/mc.exe |
```

Comment:
why are we calling this 64-bit and not 64-bit Intel?

Reply:
Old baggage..